subsys/bluetooth/controller/ticker/ticker.c

```diff
@@ -146,6 +146,14 @@ struct ticker_user_op_update {
 					 * >1: latency = lazy - 1
 					 */
 	uint8_t  force;			/* Force update */
+#if defined(CONFIG_BT_TICKER_EXT)
+	uint8_t must_expire;		/* Node must expire, even if it
+					 * collides with other nodes:
+					 *  0x00: Do nothing
+					 *  0x01: Disable must_expire
+					 *  0x02: Enable must_expire
```

Comment:
Would defines for these values make sense?

Reply:
I actually had defines, but opted for following the established convention in ticker_update, e.g. 
lazy = 0: Ignore
lazy > 0: latency = lazy - 1

Reply:
So any value > 0x1 enables must_expire right?

Reply:
Yes, that would be like setting a pseudo-boolean uint_8 to e.g. 2